$MirOS: ports/devel/cvs/patches/patch-src_root_c,v 1.1 2006/10/02 05:59:18 tg Exp $
--- src/root.c.orig	Sat Mar 10 23:14:20 2007
+++ src/root.c	Sat Mar 10 23:14:14 2007
@@ -18,6 +18,8 @@
 #include <assert.h>
 #include "getline.h"
 
+__RCSID("$MirOS: src/gnu/usr.bin/cvs/src/root.c,v 1.5 2005/12/16 16:09:46 tg Exp $");
+
 /* Printable names for things in the current_parsed_root->method enum variable.
    Watch out if the enum is changed in cvs.h! */
 
@@ -635,7 +637,7 @@ parse_cvsroot (const char *root_in)
     newroot->isremote = (newroot->method != local_method);
 
 #if defined (CLIENT_SUPPORT) || defined (SERVER_SUPPORT)
-    if (readonlyfs && newroot->isremote)
+    if (readonlyfs && newroot->isremote && (newroot->method != ext_method))
 	error (1, 0,
 "Read-only repository feature unavailable with remote roots (cvsroot = %s)",
 	       cvsroot_copy);
